packages/otelbin/src/app/og/Node.tsx

```diff
@@ -66,7 +66,7 @@ const Node = ({ data, icon, type }: { data: IData; icon: React.ReactNode; type:
 			<div style={customNodeStyles} tw="flex-col">
 				<div
 					tw={`flex w-full flex-col items-center justify-center px-2 ${
-						splitLabel[1] && splitLabel[1].length > 0 && "mt-[2px]"
+						splitLabel[1] && splitLabel[1].length > 0 ? "mt-[2px]" : ""
```

Comment:
This caused warnings in the logic because of an unknown tailwind utility called `undefined`